frontend/pages/index.tsx

```diff
@@ -30,17 +36,37 @@ export const getStaticProps: GetStaticProps = async () => {
         }
       },
       cheftext,
-      chefname
+      chefname,
+    }
+  `;
+  const galleryWidgetQuery = `
+  *[_type == "HomePage"]{
+      galleryPhotos[]{
+        asset -> {
+        url
+        }
+      },
+      menuName,
+      menuLink,
+      menuDescription[]{
+        children[]{
+          text
+        }
+      }
     }
   `;
 
-  const homePageContent = await sanityClient.fetch(query);
+  const homePageContent = await Promise.all([
+    sanityClient.fetch(heroQuery),
+    sanityClient.fetch(galleryWidgetQuery),
+  ]);
+  const [heroContent, galleryContent] = homePageContent;
 
   return {
     props: {
-      homePageContent,
+      heroContent,
+      galleryContent,
     },
-    revalidate: 86400,
+    revalidate: 60,
```

Comment:
Remove revalidate please. We just don't need ISP for hosting reason

Reply:
Sure